cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/cheddar/infrastructure/persistence/filestore/tx/TransactionalFileStore.java

```diff
@@ -25,23 +24,22 @@
 import com.clicktravel.cheddar.infrastructure.persistence.database.exception.NonExistentItemException;
 import com.clicktravel.cheddar.infrastructure.persistence.filestore.FileItem;
 import com.clicktravel.cheddar.infrastructure.persistence.filestore.FilePath;
-import com.clicktravel.cheddar.infrastructure.persistence.filestore.InternetFileStore;
+import com.clicktravel.cheddar.infrastructure.persistence.filestore.FileStore;
 import com.clicktravel.cheddar.infrastructure.tx.NestedTransactionException;
 import com.clicktravel.cheddar.infrastructure.tx.NonExistentTransactionException;
 import com.clicktravel.cheddar.infrastructure.tx.TransactionException;
 import com.clicktravel.cheddar.infrastructure.tx.TransactionalResource;
-import com.clicktravel.infrastructure.persistence.aws.s3.S3FileStore;
 
-public class TransactionalS3FileStore implements InternetFileStore, TransactionalResource {
+public class TransactionalFileStore implements FileStore, TransactionalResource {
```

Comment:
I see the contract has changed, this class no longer supports InternetFileStore. Is there such a thing as a TransactionalInternetFileStore?